schema/bom-1.5.schema.json

```diff
@@ -6,8 +6,7 @@
   "$comment" : "CycloneDX JSON schema is published under the terms of the Apache License 2.0.",
   "required": [
     "bomFormat",
-    "specVersion",
-    "version"
```

Comment:
`version` has a default value and is therefore optional.

Reply:
see also #78

Reply:
specVersion should be required.

Reply:
this is true. and `specVersion` is still in the `required` set.
only `version` was removed.